Pear/View/MeetupStatusView.swift

```diff
@@ -202,12 +202,12 @@ class MeetupStatusView: UIView {
         let fullText = NSMutableAttributedString()
             .normalFont("Oh no, it looks like your schedules don’t line up 😢 I hope it works out next time!")
 
-        if user.instagram != nil || user.facebook != nil {
-            fullText
-                .normalFont("\nYou can still reach \(user.firstName) on ")
-                .socialMediaLinks(instagram: user.instagram, facebook: user.facebook)
-                .normalFont(".")
-        }
+//        if user.instagram != nil || user.facebook != nil {
```

Comment:
Comment out for now because backend took social media out of user model but I'm requesting they add it back. Will uncomment when that is done.